paper/paper.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Vector: creating and manipulating jagged arrays of Lorentz vectors"
+tags:
+  - Python
+  - vector algebra
+  - high energy physics
+authors:
+  - name: Henry Schreiner
+    orcid: 0000-0002-7833-783X
+    equal-contrib: true
+    affiliation: 1
+  - name: Jim Pivarski
+    orcid: 0000-0002-6649-343X
+    equal-contrib: true
+    corresponding: true
+    affiliation: 1
+  - name: Saransh Chopra
+    orcid: 0000-0003-3046-7675
+    equal-contrib: true
+    affiliation: 1
+
+affiliations:
+  - name: Princeton University
+    index: 1
+date: 16 August 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+Vector algebra is a crucial component of data analysis pipelines in high energy
```

Comment:
"Vector algebra" could mean different things to different people. For me, "algebra" makes me think "abstract algebra," so I'd be looking for addition-like and multiplication-like operators with some properties like associativity or distributivity and some sense of closure. That would lead me to think it's a linear algebra library, like BLAS. That's not what you mean here!

Depending on the reader's background, "vector" can mean

* a 2D, 3D, or 4D _physical space_ vector, like what the Vector library is actually about,
* an N×1 or 1×N matrix without physical interpretation, as in ordinary linear algebra,
* an N×1 or 1×N vector or covector ("1-form") of geometric algebra, which could live in a non-Euclidean metric (as our 4D Lorentz vectors already do),
* a member of an infinite-dimensional space, like a Hilbert space, such as a quantum state represented by a bra `<x|` or ket `|x>`,
* the input or output of a machine learning model, consisting of a fixed number of features or predictions,
* the direction that an airplane is flying,
* a collection-type data structure that isn't quite an array because it has variable length, like a C++ `std::vector`,
* a collection-type data structure that isn't quite an array because it's immutable, like a Lisp vector or a vector in other functional libraries,
* an organism, object, or environmental current that carries disease from one population to another,
* a plasmid or virus that carries genetic material into a host cell in genetic engineering,
* graphic primitives that use precisely positioned elements with infinite resolution, as in SVG or PDF file formats, rather than rasterized images like PNG or JPG,
* the direction and magnitude of a literary or cultural trend, a philosophical argument or line of thought, a narrative in fiction, or an architectural design.

So you'll need to narrow in quickly and let the reader know that this is about 2D and 3D Euclidean vectors and 4D Lorentz vectors that can be used as physical quantities, such as position, momentum, and forces. Instead of "algebra," a word like "common operations" or "mathematical manipulations"?